### DISCUSSION TITLE
Problem on php@7.4 Document could not be loaded

### DISCUSSION THREAD

**Discussion by ghost** [dedoc/scramble#90] [Q&A] [answered]
<img width="918" alt="image" src="https://user-images.githubusercontent.com/102802845/205655619-e60515ad-6849-4d35-b54f-eb45db071bc9.png">

I cant access the docs page on php 7.4

My composer contains:
"dedoc/scramble": "^0.6.0",
"doctrine/dbal": "^3.5.1",
"laravel/framework": "^8.47",

It works when I use php 8.1.
Have anyone else experienced this.

**Comment by romalytvynenko** [comment, marked as answer]
@Jesper-FIltenborg-Dwarf can you try `/docs/api.json` route? There should be a real error. When you share it, I'll be able to help!

**Reply by ghost** [reply]
@romalytvynenko Thanks a lot for the advice. I found the bug myself.
There was some php 8.1 syntax in one of my resources, which broke the documentation.

But thank you for letting me know that i can use the .json route to debug.
